src/components/AdvancedSearchOptions/AdvancedSearchOptionsPresentation.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+
+// FPCC
+import SingleSelect from 'components/AdvancedSearchOptions/SingleSelect'
+import {
+  HAS_AUDIO,
+  HAS_IMAGE,
+  HAS_VIDEO,
+  HAS_TRANSLATION,
+  SORT,
+  SORT_ALPHABETICAL,
+  SORT_CREATED,
+  SORT_MODIFIED,
+  TRUE,
+  FALSE,
+} from 'common/constants'
+
+function AdvancedSearchOptionsPresentation() {
+  return (
+    <div data-testid="AdvancedSearchOptionsPresentation" className="bg-white">
+      <div className="mx-auto max-w-3xl px-4 text-center sm:px-6 lg:max-w-7xl lg:px-8">
+        <section
+          aria-labelledby="filter-heading"
+          className="border-t border-gray-200 pt-2 pb-4"
+        >
+          <h2 id="filter-heading" className="sr-only">
+            Dictionary filters
+          </h2>
+
+          <div className="flex items-center justify-between">
+            <div className="flex items-baseline space-x-8">
+              <SingleSelect
```

Comment:
Does this sort selector need a label? I know we are planning to sometimes link straight to a pre-filled form, and I wonder whether a value like "Recently created" would be ambiguous

Reply:
This didn't come up in the design conversations but I think you make a really valid point. I can implement a label or other solution when working on the Reports page. @madebybridget Design thoughts?